### ISSUE TITLE
[FSR] Rework concept for value pair list loops

### ISSUE THREAD

**Issue by rebeccawalsh** [department-of-veterans-affairs/va.gov-team#21755]
## Issue Description
In our list loop discussions with Ryan and the team last week, he asked for us to all follow up with list loops concepts for our products. Ping accessibility. Then, post to design Slack channel once the sequential version is complete, so the group can discuss. I followed up with Ryan and he suggests that we remove the edit/save functionality for our value pair version of the list loop.

---
## Tasks
- [ ] Create new value pair list loop concept and upload to Sketch for Teams
- [ ] Follow up with accessibility for review
- [ ] Post to dsva slack channel for discussion

## Acceptance Criteria
- [ ] Concept uploaded to SfT and shared to #design

**Comment by rebeccawalsh** [comment]
This work is not really relevant anymore since we will not be using list loops.
